src/states/Preroll.js

```diff
@@ -227,16 +227,21 @@ class Preroll extends AdState {
   }
 
   resumeAfterNoPreroll(player) {
+
     // Resume to content and unblock play as there is no preroll ad
     this.contentResuming = true;
     player.ads._shouldBlockPlay = false;
+    this.cleanupPartial(player);
 
     // Play the content if we had requested play or we paused on 'contentupdate'
     // and we haven't played yet. This happens if there was no preroll or if it
     // errored, timed out, etc. Otherwise snapshot restore would play.
-    if (player.paused() &&
-        (player.ads._playRequested || player.ads._pausedOnContentupdate)) {
-      player.play();
+    if (player.paused() && (player.ads._playRequested || player.ads._pausedOnContentupdate)) {
+      const playPromise = player.play();
+
+      if (playPromise) {
+        playPromise.then(() => {}).catch(() => {});
```

Comment:
We should probably copy what we do in video.js here. Not sure if this is actually an issue or not though. https://github.com/videojs/video.js/blob/master/src/js/utils/promise.js#L25

Reply:
Reasonable!